README.md

```diff
@@ -78,6 +78,8 @@ Otherwise data loss of current chunks will be incurred.  See [operations guide](
 * [Graphite](https://github.com/grafana/metrictank/blob/master/docs/graphite.md)
 * [Metadata](https://github.com/grafana/metrictank/blob/master/docs/metadata.md)
 * [Tags](https://github.com/grafana/metrictank/blob/master/docs/tags.md)
+* [Whisper data import](https://github.com/grafana/metrictank/blob/master/docs/whisper-data-import.md)
+* [Schema conversion logic](https://github.com/grafana/metrictank/blob/master/docs/schema-conversion-logic.md)
```

Comment:
these 2 are way too related to be split into separate features.
can't we just have 1 page about data importing, with a section about whisper imports, and a section about schema conversion

Reply:
Sure, I just split it because I figured it is likely that sooner or later the schema conversion logic would also be used for something other than the Whisper importer. But if that happens then we can just split it again.

Reply:
I have consolidated these two pages